sdk/wallet-adapter/packages/wallet-kit/README.md

```diff
@@ -90,3 +90,4 @@ The following wallets are known to work with the Wallet Standard:
 - **[Sui Wallet](https://docs.sui.io/devnet/explore/wallet-browser)**
 - **[Ethos Wallet](https://chrome.google.com/webstore/detail/ethos-wallet/mcbigmjiafegjnnogedioegffbooigli)**
 - **[Suiet Wallet](https://suiet.app/)**
+- **[Morphis Wallet](https://chrome.google.com/webstore/detail/morphis-wallet/heefohaffomkkkphnlpohglngmbcclhi)**
```

Comment:
Thanks for opening a PR for this! I went and tried this and it looks like the icon isn't accessible from the web. Screenshot attached.
<img width="224" alt="Screen Shot 2022-12-06 at 10 27 29 PM" src="https://user-images.githubusercontent.com/109986297/206105040-d4cd59c7-71a0-404b-8e3c-2b5ae7f768de.png">

Reply:
Thank you for the feedback. This bug was fixed in our latest release v0.0.1.3.